### PR TITLE
chore: bump helm 2 version

### PR DESCRIPTION
**Pull request by alejandroEsc**
**What type of PR is this?**

Chore

**What this PR does/ why we need it**:

To get probably the latest changes and probably the last fixes to 2.16.x we bump the version to 2.16.10
This is maintenance more than anything else. 

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Review by shaneutt — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 /gh-pages/
 tmp*
 ct_previous*
+# ignore all admin.conf files
```
Comment (samvantran) [inline .gitignore:4]: probably don't need this comment as its in the `gitignore` file

**Review by samvantran — COMMENTED** [review]
Looks like helm2 test is failing in a few PRs eg this and #774

File: .gitignore (hunk 1 above)
Comment (alejandroEsc) [reply .gitignore:4]: ha, good point.

**Review by samvantran — APPROVED** [review]

**Pull request merged**